lib/components/Information/Badge/index.tsx

```diff
@@ -1 +1,33 @@
-export * from "@/ui/badge"
+import { Avatar } from "@/components/Information/Avatar"
+import { cn } from "@/lib/utils"
+import { Badge as BadgeComponent } from "@/ui/badge"
+import { ComponentProps, forwardRef } from "react"
+
+interface BadgeProps {
+  text: string
+  hasAvatar?: boolean
+  variant?: ComponentProps<typeof BadgeComponent>["variant"]
+  imageSrc?: ComponentProps<typeof Avatar>["src"]
+  alt?: ComponentProps<typeof Avatar>["alt"]
+}
+
+export const Badge = forwardRef<HTMLDivElement, BadgeProps>(
+  ({ text, hasAvatar = false, imageSrc, alt, variant = "name" }) => {
+    return (
+      <BadgeComponent
+        variant={variant}
+        className={cn(hasAvatar ? "pl-[0.15rem] pr-2" : "px-2")}
+      >
+        {hasAvatar && (
+          <Avatar
+            alt={alt || text[0]}
+            src={imageSrc}
+            size="xxsmall"
+            className="mr-1 bg-orange-500"
```

Comment:
Why `bg-orange-500` in this specific instance? Can we solve this in a different way? Should this background color be on the avatar itself?

Reply:
I wasn't sure about this one... When there's no specific avatar it will take the first letter of the name and create the avatar. I don't know if the background color is random or there's a specific color for each team for example. In any case if you send an image source it will override this and just show the image, this is more of a fallback mechanism.

Reply:
I feel like this logic should go in the `AvatarFallback` component then, no?